### PR TITLE
Unable button not working

### PR DESCRIPTION
**Pull request by F-Naddaf**

### CODE REVIEW

File: src/pages/questionPage.js [hunk 1]
```diff
@@ -2,32 +2,48 @@
...
-  const questionElement = createQuestionElement(currentQuestion.text);
-  userInterface.appendChild(questionElement);  
+  
+  const questionElement = createQuestionElement(currentQuestion.text)
+  userInterface.appendChild(questionElement); 
+  
+  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
 
-  const answerElement = createAnswerElements();
+  const unableTheButton = document.getElementById(NEXT_QUESTION_BUTTON_ID)
```
Comment (jontra000) [inline src/pages/questionPage.js:24]: You should use SKIP_THE_QUESTION here, not NEXT_QUESTION_BUTTON_ID

File: src/views/answerView.js [hunk 2]
```diff
@@ -1,34 +1,32 @@
...
-
-    let correctAnswerScore = document.createElement('span')
-    correctAnswerScore.textContent = 0;
-    const correctAnswerMessage = document.createElement('div')
-    correctAnswerMessage.classList.add('message')
-    correctAnswerMessage.textContent = `You have answered ${score} question correct`
-
-  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
-  let answerClicked = false;
+ export const createAnswerElements = (answersListElement, unableTheButton) => {
```
Comment (jontra000) [inline src/views/answerView.js:12]: `unableTheButton` should be called `skipButton` to make it easier to understand the code.

**Review by jontra000 — CHANGES_REQUESTED** [review]
Just a bug and a couple of small things to fix

File: src/pages/questionPage.js (hunk 1 above)
Comment (F-Naddaf) [reply src/pages/questionPage.js:24]: This one here to unable the next question button not to skip the question that's why I used NEXT_QUESTION_BUTTON_ID

Comment (F-Naddaf) [reply src/pages/questionPage.js:24]: because I give attribute disable for the next question button.

Comment (jontra000) [reply src/pages/questionPage.js:24]: Oh yes, I see - in that case the variable should be called `nextQuestionButton`.

**Pull request merged**